Mage.Sets/src/mage/cards/g/GalvanicGiant.java

```diff
@@ -0,0 +1,59 @@
+package mage.cards.g;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.SpellCastControllerTriggeredAbility;
+import mage.abilities.effects.common.DrawDiscardControllerEffect;
+import mage.abilities.effects.common.TapTargetEffect;
+import mage.abilities.effects.common.counter.AddCountersTargetEffect;
+import mage.constants.SubType;
+import mage.counters.CounterType;
+import mage.filter.FilterSpell;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.target.common.TargetOpponentsCreaturePermanent;
+import mage.cards.AdventureCard;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class GalvanicGiant extends AdventureCard {
+
+    private static FilterSpell filter = new FilterSpell("a spell with mana value 5 or greater");
+
+    {
+        filter.add(new ManaValuePredicate(ComparisonType.MORE_THAN, 4));
```

Comment:
It is equivalent, but there is `new ManaValuePredicate(ComparisonType.OR_GREATER, 5)` to be closer to the rules text.